### PR TITLE
Add @nimishbongale's first blog post

### PR DESCRIPTION
**Pull request by nimishbongale**
## Description
Adds @nimishbongale's blog post to the open source site, which talks about vocabulary and the other relevant details.

## Screenshots
screencapture-localhost-5000-blog-entries-cc-vocabulary-docs-intro-2020-10-02-16_40_58

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: content/blog/series/gsod-2020-vocabulary-usage-guide/contents.lr [hunk 1]
```diff
@@ -0,0 +1 @@
+name: GSOD-2020: Vocabulary Usage Guide
```
Comment (dhruvkb) [inline content/blog/series/gsod-2020-vocabulary-usage-guide/contents.lr:1]: I'm afraid the second colon will be misinterpreted unless we quote it. I could be wrong.

```suggestion
name: "GSoD-2020: Vocabulary Usage Guide"
```

File: content/blog/entries/cc-vocabulary-docs-intro/contents.lr [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+### What drives me
+Documentation is one of the primary reasons which determines how successful a certain open source library will be. The major question that developers think of while choosing a suitable tech stack to build their applications is:
+
+- Is the library _well documented_? 
+- Is it _well maintained_? 
+- Does it have some _considerable usage and error support_?
+
+These are exactly the questions I should be asking myself while going about this project idea.
+
+As aforementioned, there is an immanent need to have a concise and consolidated documentation. The lack of documentation hurts the future perspectives of open source applications, and is by far, an essential and non-negligible component. Linking to these documentations should be an appealing home page, which captures the interest of the people in an instant. The documentation should be well organised, thereby enabling a seamless flow through it.
```
Comment (dhruvkb) [inline content/blog/entries/cc-vocabulary-docs-intro/contents.lr:37]: *imminent 😄

**Review by dhruvkb — APPROVED** [review]

File: content/blog/series/gsod-2020-vocabulary-usage-guide/contents.lr (hunk 1 above)
Comment (nimishbongale) [reply content/blog/series/gsod-2020-vocabulary-usage-guide/contents.lr:1]: Oh, okay alright!

File: content/blog/entries/cc-vocabulary-docs-intro/contents.lr (hunk 2 above)
Comment (nimishbongale) [reply content/blog/entries/cc-vocabulary-docs-intro/contents.lr:37]: I think I mean _immanent_ here if I'm not mistaken 🤔

imminent = obvious & approaching
immanent = inherent / innate

> There is an inherent need...

This is what I wanted it to mean, but turns out even if I replace `immanent` with `imminent` the sentence would still make sense, but in a very different way 😄

**Pull request merged**